airflow/example_dags/example_docker_operator.py

```diff
@@ -17,6 +17,7 @@
 # specific language governing permissions and limitations
 # under the License.
 """
+import airflow
 from airflow import DAG
 from airflow.operators import BashOperator
 from datetime import datetime, timedelta
```

Comment:
we don’t need to import “datetime” here anymore I think?

Reply:
suggest to change to “from datetime import timedelta”.

I sent my laptop to repair and using iPad checking the file, so bear with me if I missed checking anything ;)

Reply:
You are right but actually the whole content of this file is a single comment `""" """`, because docker is a optional dependency I guess. Could this be the reason? 
All other example dags would work without any extras.

Reply:
Get it. But it may still be worthwhile to address this as it’s gonna be a sample dag and the change is quite minimal.
Happy to merge after it’s addressed

Reply:
The same suggestion for the docker_copy_data.py as well

Reply:
Done. :)